index.bs

```diff
@@ -645,7 +645,7 @@ steps <a>in parallel</a>:
   2. Check that the algorithm was triggered while the [=relevant global object=]
      had a <a>transient activation</a>. Otherwise, <a>reject</a> |promise| with
      a {{SecurityError}} and abort these steps.
-  3. Set <code>|status|.{{PermissionStatus/state}}</code> to `"ask"`.
+  3. Set <code>|status|.{{PermissionStatus/state}}</code> to <code>"ask"</code>.
```

Comment:
Instead of this, you could add

```
Markup Shorthands: markdown yes
```

to the initial `<pre class="metadata">`. The [last time I thought about a complete configuration](https://github.com/WICG/webpackage/blob/main/loading.bs), I used

```
Complain About: accidental-2119 yes, missing-example-ids yes
Markup Shorthands: markdown yes, css no
Assume Explicit For: yes
```

Reply:
I was surprised I didn't have Markdown enabled but then I remembered that I had gotten thoroughly frustrated by it at some point earlier when editing the specification and made the decision to turn it off. Let me land this as is and then consider whether I want to do that refactoring.